dd-sdk-android/src/main/kotlin/com/datadog/android/webview/internal/log/WebViewLogEventConsumer.kt

```diff
@@ -38,8 +38,8 @@ internal class WebViewLogEventConsumer(
         }
     }
 
-    private fun map(event: JsonObject): JsonObject? {
-        try {
+    private fun map(event: JsonObject): JsonObject {
+        return try {
```

Comment:
I'm not a fan of using this syntax. The `return` encompass a large try/catch block where the return value is always the same, so it doesn't make sense to do that. 
Also most of the caught exceptions are for a single line, so maybe only the unsafe line should be within the try-catch block instead of the whole method.

Reply:
Regarding last remark, not necessarily as the `addTags` function can also throw.

Reply:
It can throw but not the same exceptions